app/components/RouteHistory.tsx

```diff
@@ -71,6 +79,12 @@ const RouteHistory = ({ history }: Props) => {
     return formatAmountAndAsset(tx.outputAmount, tx.outputAsset, tx, DirectionType.OUTPUT);
   };
 
+  const priceOf = (tx: RouteHistoryAlias): string => {
+    return formatPrice(tx.inputAmount,tx.outputAmount,tx.inputAsset, tx.outputAsset, tx);
+  };
```

Comment:
You use 5 parameters here, but `formatPrice` only has 3? Maybe you wanted to add assets to the formatted price (which would make sense)?